homeassistant/components/upnp/config_flow.py

```diff
@@ -160,6 +161,10 @@ async def async_step_ssdp(self, discovery_info: Mapping) -> Mapping[str, Any]:
         """
         _LOGGER.debug("async_step_ssdp: discovery_info: %s", discovery_info)
 
+        if self.hass.data.get(DOMAIN, {}).get(DOMAIN_IGNORE_DISCOVERIES, False):
```

Comment:
Is it possible to check if IP address is configured and abort on that?

It's annoying if we have to ask every user to implement the same flag because they are getting spammed.

Reply:
Great idea, thank you!

Not sure if we should only check for configured config entries or also ignored/unconfigured config entries. Possibly there are some other scenario's where this might not fully work, but it should already be a step up from the current problems.

Reply:
Updated, please review.